Validation/Geometry/src/MaterialBudgetHGCalHistos.cc

```diff
@@ -284,6 +317,39 @@ void MaterialBudgetHGCalHistos::book()
 
+  // Epoxy
+  hmgr->addHistoProf1( new TProfile("11110", "IL prof Eta [Epoxy];#eta;#lambda/#lambda_{0}", 250, -5.0, 5.0 ) );
+  hmgr->addHistoProf1( new TProfile("11120", "IL prof Phi [Epoxy];#varphi [rad];#lambda/#lambda_{0}", 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf2( new TProfile2D("11130", "IL prof Eta  Phi [Epoxy];#eta;#varphi;#lambda/#lambda_{0}", 250, -5., 5., 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf1( new TProfile("11140", "IL prof R [Epoxy];R [mm];#lambda/#lambda_{0}", 300, 0., 3000. ) );
+  hmgr->addHistoProf2( new TProfile2D("11150", "IL prof sum R  z [Epoxy];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11152", "IL ortho prof sum R  z [Epoxy];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHisto2( new TH2F("11160", "IL prof local R  z [Epoxy];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11170", "IL prof local R  z [Epoxy];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11172", "IL ortho prof local R  z [Epoxy];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+
+  // Kapton
+  hmgr->addHistoProf1( new TProfile("11210", "IL prof Eta [Kapton];#eta;#lambda/#lambda_{0}", 250, -5.0, 5.0 ) );
+  hmgr->addHistoProf1( new TProfile("11220", "IL prof Phi [Kapton];#varphi [rad];#lambda/#lambda_{0}", 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf2( new TProfile2D("11230", "IL prof Eta  Phi [Kapton];#eta;#varphi;#lambda/#lambda_{0}", 250, -5., 5., 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf1( new TProfile("11240", "IL prof R [Kapton];R [mm];#lambda/#lambda_{0}", 300, 0., 3000. ) );
+  hmgr->addHistoProf2( new TProfile2D("11250", "IL prof sum R  z [Kapton];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11252", "IL ortho prof sum R  z [Kapton];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHisto2( new TH2F("11260", "IL prof local R  z [Kapton];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11270", "IL prof local R  z [Kapton];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+  hmgr->addHistoProf2( new TProfile2D("11272", "IL ortho prof local R  z [Kapton];z [mm];R [mm];x/X_{0} ", nzbin, zMin, zMax, nrbin, rMin, rMax ) );
+
+  // Aluminium
+  hmgr->addHistoProf1( new TProfile("11310", "IL prof Eta [Aluminium];#eta;#lambda/#lambda_{0}", 250, -5.0, 5.0 ) );
+  hmgr->addHistoProf1( new TProfile("11320", "IL prof Phi [Aluminium];#varphi [rad];#lambda/#lambda_{0}", 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf2( new TProfile2D("11330", "IL prof Eta  Phi [Aluminium];#eta;#varphi;#lambda/#lambda_{0}", 250, -5., 5., 180, -3.2, 3.2 ) );
+  hmgr->addHistoProf1( new TProfile("11340", "IL prof R [Aluminium];R [mm];#lambda/#lambda_{0}", 300, 0., 3000. ) );
```

Comment:
Magic numbers (300, 3000, 250, etc.) should be replaced by constants, like in line 347 that uses `zMin`, `zMax`, and so on.